### DISCUSSION TITLE
Copy/Pasting VSTs and effects in the mixer

### DISCUSSION THREAD

**Discussion by RyanK1729** [musescore/MuseScore#14544] [Ideas]
Relatively straight forward. It would be nice to be able to copy a VST and its settings or the effects and their settings from one instrument in the mixer to another, as oppose to having to recreate everything from scratch for each instrument. Would save a fair bit of time when working with large scores.

**Comment by randoguyname** [comment]
Perhaps this could be implemented like it is in the Logic Pro X mixer, where dragging the effect can move it from one channel to another, and holding alt while dragging the effect copies it.

**Comment by SparkFountain** [comment]
I would like to bring up this topic again for MuseScore 4, since it's very tedious to click through all sub-menus of the VST instrument selection for every part again.

I usually work with scores for voices, with up to 8 or 9 parts. Since there are many new instrument sounds available, I simply try them out for one part - because it's so time-consuming to select an instrument eight or nine times again, just to discover it's not what you are looking for...

**Specific Feature Request:**
Please add a kind of "Copy-Paste" function for a part setting in the mixer. So that it can easily be applied 1:1 to other parts in the mixer. Or maybe even a checkbox / button etc. labeled "apply for all parts", so that you only define the whole setting once.

Would really appreciate this feature implementation! 💚

**Comment by scorster** [comment]
Agreed. Hierarchic menus are generally pretty awkward to navigate. MuseScore 4's Mixer brought that to a new pinnacle. Unfortunately MuseScore seems pointlessly allergic to dialogs.

### Also see a similar request here: 

https://github.com/musescore/MuseScore/issues/24686
